src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -173,6 +195,7 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
                 SetResolveData(resultId, completionList);
             }
 
+            _logger.LogInformation("Returning final completion list.");
```

Comment:
What does final mean in this case?

Reply:
Updated logging to remove `final`. 

Essentially we were "returning" in a few places (pending post-processing), and this represented the "final" return to LSP platform. Updated verbiage to say "found xyz" elsewhere and returning here.